src/main/webapp/Portal.html

```diff
@@ -125,8 +125,8 @@
 			head.load(cdnEndpoint + "generated/react-jsonschema-form.js");
 			head.load(cdnEndpoint + "generated/universalCookie.min.js");
 			head.load(cdnEndpoint + "generated/react-bootstrap.min.js");
+			head.load(cdnEndpoint + "generated/sanitize-html.min.js");
 			head.load(cdnEndpoint + "generated/synapse-react-client.production.min.js");
-			head.load(cdnEndpoint + "js/sanitize-html.min.js");
```

Comment:
sanitize-html is now retrieved via yarn...where is this used so we can validate that this doesn't break whatever was using the old version?

Reply:
I'd verify a few places.  As a user who is not on the special js/html/css team, try uploading an html file that has js, and verify that the preview looks ok.  But SynapseJSNIUtilsImpl.sanitizeHtml() uses [xss](https://www.npmjs.com/package/xss), which is still loaded from the js directory, so I think sanitize-html was just being loaded for SRC.